app/assets/javascripts/code/services/google/info_window.js

```diff
@@ -8,22 +8,26 @@ angular.module("google").factory("infoWindow", ["map", "$http", "$compile", "$ro
     get: function() {
       return this.popup;
     },
-    show: function(url, data, position){
-      var self = this;
+
+    show: function(url, data, position, type){
       this.popup.setContent("working..");
       this.popup.setPosition(position);
       this.popup.open(map.get());
-      $timeout(function(){
-        $http.get(url, {params : data, cache: true}).success(_(self.onShowData).bind(self));
+      const htmlFile = (type === "Fixed") ? '/partials/fixed_info_window.html' : '/partials/info_window.html';
```

Comment:
You could extract the file paths on line 4 into constants (eg `const FIXED_PATH`)

Reply:
Have you considered a different name for `htmlFile`? I don't like it much cause it's not really a file. Maybe `partialPath` or `htmlPath`?